command/update/command.go

```diff
@@ -117,112 +113,48 @@ func (c *Command) Execute(cmd *cobra.Command, args []string) {
 func (c *Command) execute() error {
 	var err error
 
+	var k8sClient kubernetes.Interface
+	{
+		k8sConfig := k8s.DefaultConfig()
+
+		k8sConfig.Address = f.Kubernetes.Address
+		k8sConfig.Logger = microloggertest.New()
+		k8sConfig.InCluster = f.Kubernetes.InCluster
+		k8sConfig.TLS.CAFile = f.Kubernetes.TLS.CaFile
+		k8sConfig.TLS.CrtFile = f.Kubernetes.TLS.CrtFile
+		k8sConfig.TLS.KeyFile = f.Kubernetes.TLS.KeyFile
+
+		k8sClient, err = k8s.NewClient(k8sConfig)
+		if err != nil {
+			return microerror.MaskAny(err)
+		}
+	}
```

Comment:
I wanted to cleanup how the endpoint updater is executed in the kvm-operator. It was easier to use the operatorkit way of creating a new k8s client like we do it elsewhere these days.